src/poc/miner_poc_mgr_db_owner.erl

```diff
@@ -0,0 +1,189 @@
+%%%-------------------------------------------------------------------
+%% @doc
+%% == poc mgr db owner and related functions ==
+%%
+%% * This process is started first in the miner supervision tree
+%% * POC mgr will get the db reference from here when they init
+%% * This process also traps exits and closes rocksdb (if need be)
+%% * This process is responsible for serializing local POC updates to disk in a
+%%   batch write each write interval (currently 1000 millis)
+%% * local POCs are POC which are running and active on this validator
+%%
+%% @end
+%%%-------------------------------------------------------------------
+-module(miner_poc_mgr_db_owner).
+
+-behavior(gen_server).
+
+%% api exports
+-export([start_link/1,
+         db/0,
+         poc_mgr_cf/0,
+         write/2,
+         gc/1
+        ]).
+
+%% gen_server exports
+-export([
+    init/1,
+    handle_call/3,
+    handle_cast/2,
+    handle_info/2,
+    terminate/2,
+    code_change/3
+]).
+
+-define(DB_FILE, "poc_mgr.db").
+-define(TICK, '__poc_write_tick').
+
+-record(state, {
+          db :: rocksdb:db_handle(),
+          default :: rocksdb:cf_handle(),
+          poc_mgr_cf :: rocksdb:cf_handle(),
+          write_interval = 1000 :: pos_integer(),
+          tref :: reference(),
+          pending = #{} :: maps:map()
+         }).
+
+%% api functions
+start_link(Args) ->
+    gen_server:start_link({local, ?MODULE}, ?MODULE, Args, []).
+
+-spec db() -> rocksdb:db_handle().
+db() ->
+    gen_server:call(?MODULE, db).
+
+-spec poc_mgr_cf() -> rocksdb:cf_handle().
+poc_mgr_cf() ->
+    gen_server:call(?MODULE, poc_mgr_cf).
+
+-spec write( POC :: miner_poc_mgr:local_poc(),
+             Skewed :: skewed:skewed()) -> ok.
+write(POC, Skewed) ->
```

Comment:
Is this still used?

Reply:
hoping to move these to the ledger post merge